components/service/glean/src/main/java/mozilla/components/service/glean/Glean.kt

```diff
@@ -229,6 +229,7 @@ open class GleanInternalAPI internal constructor () {
 
         // Set up information and scheduling for glean owned pings
         metricsPingScheduler = MetricsPingScheduler(applicationContext)
+        metricsPingScheduler.startupCheck()
```

Comment:
nit: Maybe these three lines belong in a new function `initializePings`, since these don't really initialize metrics values?